src/questdb/extra_cpython.pxd

```diff
@@ -27,6 +27,10 @@ cdef extern from "Python.h":
     str PyUnicode_FromStringAndSize(
         const char* u, Py_ssize_t size)
 
+    # Ditto, see comment on why not returning a `PyObject` above.
+    object PyBytes_FromStringAndSize(
+        const char* u, Py_ssize_t size)
```

Comment:
Remove.

This is already defined in the cython standard lib correctly.
See: https://github.com/cython/cython/blob/master/Cython/Includes/cpython/bytes.pxd where it defines:


```cython
bytes PyBytes_FromStringAndSize(char *v, Py_ssize_t len)
```

Usage within `ingress.pyx` would be something like:

```cython
from cpython.bytes cimport PyBytes_FromStringAndSize
```